config.toml.example

```diff
@@ -373,7 +371,9 @@ changelog-seen = 2
 
 # Sets the number of codegen units to build the standard library with,
 # regardless of what the codegen-unit setting for the rest of the compiler is.
-#codegen-units-std = 1
+# NOTE: building with anything other than 1 is known to occasionally have bugs.
+# See https://rust-lang.zulipchat.com/#narrow/stream/131828-t-compiler/topic/bootstrapping.20on.20s390x.2Fmips.20for.20musl/near/227436952 for details.
```

Comment:
I would prefer we not link to ephemeral chats like this. If there's some useful conversation there, we can either inline the salient points into this file or otherwise document it (e.g., on rustc-dev-guide) and link there. One other approach would be to file a GitHub issue with some details on the bugs encountered and how to reproduce them and say something like "those bugs are tracked here: #xxx". If it's true that just setting codegen-units-std to non-1 value causes bugs for most folks we should probably prioritize an investigation at least.

Reply:
Ok, I opened https://github.com/rust-lang/rust/issues/83600 and I can link there in config.toml.